.ci/common-ci-settings-and-functions.sh

```diff
@@ -1,97 +0,0 @@
-#!/bin/bash
```

Comment:
none of the scripts in `.ci` folder will be used, because we're going to use [centralized scripts](https://github.com/openshift/release/pull/34366/files#diff-57b69af98e754326930884620124ca4865eaefcae241e7b3223d8d596c89c86a)